devices/osram.js

```diff
@@ -83,7 +83,7 @@ module.exports = [
         model: 'AC03642',
         vendor: 'OSRAM',
         description: 'SMART+ CLASSIC A 60 TW',
-        extend: extend.ledvance.light_onoff_brightness_colortemp(),
+        extend: extend.ledvance.light_onoff_brightness_colortemp({colorTempRange: [153, 370], disableColorTempStartup: true}),
```

Comment:
disableColorTempSupport can be removed since it is already disabled by default now: https://github.com/Koenkk/zigbee-herdsman-converters/blob/ba4a9c7f5d3eba5ff9939608fcfd6b600d23840d/lib/extend.js#L136